pyproject.toml

```diff
@@ -62,7 +62,7 @@ test = [
   "types-pytest-lazy-fixture>=0.6.3.3",
 ]
 dev = ["tox", "isort", "black"]
-mypy = ["mypy", "types-requests", "types-click"]
+mypy = ["mypy", "types-requests"]
```

Comment:
`types-click` was actually breaking mypy for me locally - I suppose it's no longer updated with click v8 (hence pinning above)